api/src/handlers/types.rs

```diff
@@ -23,6 +33,12 @@ pub struct ScarbCompileResponse {
     pub file_content_map_array: Vec<FileContentMap>,
 }
 
+impl Successable for ScarbCompileResponse {
+    fn is_successful(&self) -> bool {
+        self.message == "Success"
```

Comment:
```suggestion
        self.message.to_lower_case() == "success"
```

Reply:
Well, our responses need to be changed in general to follow restful practices. This was done in horizontal scaling work on zksync side, but it's abandoned now. Anyway, it is a whole separate issue

Reply:
Seems redundant, better to standardize responses